pkg/api/deploymentapi/search.go

```diff
@@ -32,8 +34,9 @@ import (
 type SearchParams struct {
 	*api.API
 
-	Context context.Context
-	Request *models.SearchRequest
+	Context         context.Context
+	Request         *models.SearchRequest
+	MinimalMetadata *[]string
```

Comment:
```suggestion
	MinimalMetadata []string
```

can be used and later
```go
if len(params.MinimalMetadata) > 0
```

On one hand `*[]` is not great Go practice. On the other, a pointer signifies better that this an optional parameter that will be ignored if not specified. However, I would still favor `[]string`, callers can still initialize using:
```go
SearchParams{
    ...
    MinimalMetadata: nil,
    ...
}

Reply:
Yeah, good point :+1: